### PR TITLE
BREAKING(encoding/unstable): move `base64url-stream` module to `unstable-base64url-stream`

### PR DESCRIPTION
**Pull request by iuioiua**
Towards #5920

### CODE REVIEW

File: encoding/unstable_base64url_stream.ts [hunk 1]
```diff
@@ -4,6 +4,19 @@
 /**
  * Utilities for encoding and decoding to and from base64url in a streaming manner.
  *
+ * ```ts
+ * import { assertEquals } from "@std/assert";
+ * import { encodeBase64Url } from "@std/encoding/base64url";
```
Comment (kt3k) [inline encoding/unstable_base64url_stream.ts:9]: This doesn't seem used?

**Review by kt3k — COMMENTED** [review]
Left one comment

File: encoding/unstable_base64url_stream.ts (hunk 1 above)
Comment (iuioiua) [reply encoding/unstable_base64url_stream.ts]: Fixed

**Review by kt3k — APPROVED** [review]
LGTM

**Pull request merged**